### PR TITLE
Fix outdated "how to use" links

### LINKED ISSUE #2013: Static links to page `how-to-use-our-world-in-data`, which is a redirect
## Description

We have three places in code where we link out to the page `how-to-use-our-world-in-data`, which no longer exists and now links out to `faqs`:
- In the page Footer (`SiteFooter.tsx`), a "How to use" link
- At the end of every article and every Gdoc article (`LongFormPage.tsx` and `OwidArticle.tsx`), linking to `/how-to-use-our-world-in-data#how-to-embed-interactive-charts-in-your-article`, where the anchor doesn't exist in the FAQ

## Solution

Either:
- Change the links to point to the correct page
- Or, create a _more specific_ FAQ page that is more about "How to use OWID" than our current, general FAQ

### PR DESCRIPTION
**Pull request by rakyi**
These are the only places I found that referenced that old page(s?).

Fixes #2013.

### CODE REVIEW

File: site/SiteFooter.tsx [hunk 1]
```diff
@@ -89,10 +89,10 @@ export const SiteFooter = (props: SiteFooterProps) => (
                             </li>
                             <li>
                                 <a
-                                    href="/about/how-to-use-our-world-in-data"
+                                    href="/faqs"
                                     data-track-note="footer_navigation"
                                 >
-                                    How to use
+                                    FAQs
```
Comment (marcelgerber) [inline site/SiteFooter.tsx:95]: I think it's nicer to write this out as `Frequently asked questions`

**Review by marcelgerber — APPROVED** [review]
Nice. One small change if you agree; otherwise this is good to go :shipit:

File: site/SiteFooter.tsx (hunk 1 above)
Comment (rakyi) [reply site/SiteFooter.tsx:95]: The link is `FAQs` on the about page, so I opted for consistency and also there isn't much space in the footer column, so the longer link wraps on a medium screen. Do you feel strongly that we shouldn't use the acronym?

Comment (marcelgerber) [reply site/SiteFooter.tsx:95]: You're right, makes sense. Good to go ahead then :)

**Pull request merged**